examples/live_tiffs.py

```diff
@@ -0,0 +1,79 @@
+"""
+Loads and Displays tiffs as they get generated in the specific directory.
+Trying to simulate the live display of data as it gets acquired by microscope. 
+"""
+
+import os
+import sys
+import time
+import numpy as np
+from skimage.io.collection import alphanumeric_key
+from dask import delayed
+import dask.array as da
+from tifffile import imread
+import napari
+from napari.qt import thread_worker
+
+
+with napari.gui_qt():
+    viewer = napari.Viewer(ndisplay=3)
+    # pass a directory to monitor or it will monitor current directory.
+    path = sys.argv[1] if len(sys.argv) > 1 else '.' 
+    path = os.path.abspath(path)
+    end_of_experiment = 'final.log'
```

Comment:
I think most people won't have a handy "final" file that is generated (eg: that's not typical for data generated on any of the instruments in our lab at all). When I wrote a directory watcher for napari I made sure that the user could cancel watching with Ctrl+C in the terminal. That might be a nice thing to add to this example as well.

Reply:
This is just an example we can have multiple way of how we treminate the program potentially want the way to stop the `while: True` I am adding that in my next commit.  The `final.log` file is generated by the `live_tiffs_generator.py` at the moment. But, yes you are write that we can have user interupted way to do that as well. Ctrl + C sound terminating the programs and I dont think I as a user want to terminate the programs after the aqusition I rather look at the data and once all done I will close the program.

Reply:
@GenevieveBuckley one reason for this is that (a) we couldn't open the last file because it might still be getting data written to it (so the program was crashing), so (b) we would never open the final file in a series this way. It's not a huge deal, but this was the simplest thing to check for. Another option is to check the modification time for each file and only add it to the list if it hasn't been touched for, say, 1s.

Reply:
I'm ok with adding that logic in a future PR or adding it in this one, @zeroth

Reply:
@jni well I think this example makes if simple to understand. so maybe we can make it robust once we have some better way of checking the file state.